--- pjmedia/src/pjmedia-audiodev/coreaudio_dev.m
+++ pjmedia/src/pjmedia-audiodev/coreaudio_dev.m
@@ -637,11 +637,6 @@
         }
         if (cdi->info.output_count > 0) {
             cdi->info.caps |= PJMEDIA_AUD_DEV_CAP_OUTPUT_LATENCY;
-            addr.mSelector = kAudioDevicePropertyVolumeScalar;
-            addr.mScope = kAudioDevicePropertyScopeOutput;
-            if (AudioObjectHasProperty(cdi->dev_id, &addr)) {
-                cdi->info.caps |= PJMEDIA_AUD_DEV_CAP_OUTPUT_VOLUME_SETTING;
-            }
         }
         if (cf->has_vpio) {
             cdi->info.caps |= PJMEDIA_AUD_DEV_CAP_EC;
@@ -1858,11 +1853,6 @@
     {
         pi->flags |= PJMEDIA_AUD_DEV_CAP_OUTPUT_LATENCY;
     }
-    if (ca_stream_get_cap(s, PJMEDIA_AUD_DEV_CAP_OUTPUT_VOLUME_SETTING,
-                          &pi->output_vol) == PJ_SUCCESS)
-    {
-        pi->flags |= PJMEDIA_AUD_DEV_CAP_OUTPUT_VOLUME_SETTING;
-    }
     if (ca_stream_get_cap(s, PJMEDIA_AUD_DEV_CAP_INPUT_ROUTE,
                           &pi->input_route) == PJ_SUCCESS)
     {
